src/events/CallResponse.ts

```diff
@@ -0,0 +1,63 @@
+import { AttachmentBuilder } from 'discord.js'
+import type { ArgsOf } from 'discordx'
+import { Discord, On } from 'discordx'
+import * as path from 'path'
+
+// Class for simple call/response type things
+
+type CallResponse = {
+  call: string
+  response?: string
+  responseAttachment?: string
+  cooldown: number // seconds
+  lastUse: number
+  caseSensitive: boolean
+}
+
+@Discord()
+abstract class CallAndResponder {
+  private calls: CallResponse[] = [
+    {
+      call: 'tuturu',
+      responseAttachment: path.join(__dirname, '../../assets/Tuturu.png'),
+      cooldown: 60,
+      lastUse: 0,
+      caseSensitive: false,
+    },
+    {
+      call: 'zuzuru',
+      responseAttachment: path.join(__dirname, '../../assets/Zuzuru.png'),
+      cooldown: 60,
+      lastUse: 0,
+      caseSensitive: false,
+    },
+  ]
+
+  @On('messageCreate')
+  private onMessage([message]: ArgsOf<'messageCreate'>) {
+    const prompt = message.content
+    const lcPrompt = prompt.toLowerCase()
+    for (let i = 0; i < this.calls.length; i++) {
```

Comment:
I think it would be better to use `Array.prototype.forEach()` here, but the rest looks good.